CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+# Release notes 1.1.0
+
+## What's New
+
+* Experimental support for sdk based flow-control
+* Config change for multiple underlays
+
+## SDK Flow Control
+
+If the router being connected to supports it, the sdk can now manage flow control 
+instead of delegating that to the router. This is mostly importing when running
+multiple simultaneous circuits throught the SDK. When running multiple circuits,
+a slow circuit can get stalled at the router because of flow control back-pressure.
+This then back-pressures all circuits from the SDK to that router. 
+
+By moving the flow-control to the SDK, a slow circuit will not negatively other 
+circuits to the same router. This is currently enabled in the `DialOptions`.
+
+```
+t := true
+dialOptions := &ziti.DialOptions{
+    ConnectTimeout: wf.ConnectTimeout,
+    SdkFlowControl: &t,
+}
+```
+
+As this is an experimental feature, the configuration may change or be removed
+in the future.
+
+## Config Changes
+
+The multi-underlay configuration has changed. There are now two settings.
+
+```
+// If set to a number greater than one, the sdk will attempt to create multiple connections to edge routers.
+// This configuration value should not be considered part of the stable API yet. It currently defaults to one,
+// but it may default to a larger number at some point in the future or be removed. If set to zero, it will
+// be reset to one.
+MaxDefaultConnections uint32 `json:"-"`
+
+// If set to a number greater than zero, the sdk will attempt to create one or more separate connection to
+// each edge routers for control plane data, such as dials. This configuration value should not be considered
+// part of the stable API yet. It currently defaults to zero, but it may default to 1 at some point in the future
+// or be removed.
+MaxControlConnections uint32 `json:"-"`
```

Comment:
If this sticks around, we should probably add router-side configuration to limit the number of connections from a single ...identity/api session/whatever-makes-sense.

Reply:
Yes, I agree.